Directory.Build.props

```diff
@@ -3,6 +3,12 @@
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
   </PropertyGroup>
 
+  <!-- Properties that need to be available early for SDK targets -->
+  <PropertyGroup Condition="'$(MSBuildProjectName)' == 'Moq.Analyzers'">
+    <Description>Roslyn analyzer that helps to write unit tests using Moq mocking library by highlighting typical errors and suggesting quick fixes. Port of Resharper extension to Roslyn. Find the full list of detected issues at project GitHub page.</Description>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules><!-- Resolves RS1036 -->
```

Comment:
Is this duplicative now with what's been rebased in from #440? As in, #440 already fixed this so it can be removed?